signals/commands/things/src/test/java/org/eclipse/ditto/signals/commands/things/modify/DeleteAttributeTest.java

```diff
@@ -35,6 +38,7 @@
 public final class DeleteAttributeTest {
 
     private static final JsonPointer KNOWN_JSON_POINTER = JsonFactory.newPointer("key1");
+    private static final JsonPointer INVALID_JSON_POINTER = JsonFactory.newPointer("key1/äöü/foo");
```

Comment:
Shouldn't TestConstants.INVALID_JSON_POINTER be used instead? (Same value)

Reply:
Yes!